### PR TITLE
docs: add `zh-CN` translation

### PR DESCRIPTION
**Pull request by vanchKong**
commit about:
- add `zh-CN` translation, synced to `1.0.0-rc.29`
- ~~Add markdown basic usage documentation~~

translation details:
- At present, almost all translations are completed
- ~~I'll follow up with updates to the docs （my own repository: [vanchKong/VitePressCN](https://github.com/vanchKong/VitePressCN) & [site](https://vanchkong.github.io/VitePressCN/)）~~
- mostly machine translated
- A few add personal understanding
- The rest are unable to find a suitable translation，some are the original text, and some are displayed attaching the original text
- some work refers to this guy [awxiaoxian2020](https://github.com/awxiaoxian2020)

preview link
https://deploy-preview-2249--vitepress-docs.netlify.app/

### CODE REVIEW

**Comment by CHOYSEN** [comment]
refs https://github.com/vuejs/vitepress/issues/1285

**Comment by brc-dd** [comment]
@awxiaoxian2020 Let's keep both open for now. We'll decide once we reach beta.

**Comment by awxiaoxian2020** [comment]
I don't think we need two translation PRs because our goal is the same one. @CHOYSEN Could you please add me and @shellRaining collaborate your repo so that we can opt the translations conveniently?

I will close my PR at an appropriate time.

**Comment by brc-dd** [comment]
Okay sure. Yeah it will be better if you all can collaborate together. Initially I was thinking to cherry pick stuff from both. So yeah if we can have one PR with best of both, then that would be great.

**Comment by awxiaoxian2020** [comment]
Here no response from @vanchKong for time being, so I update my PR with his content. Welcome to review [it](https://github.com/vuejs/vitepress/pull/1593).

**Comment by taozuhong** [comment]
could use zh-Hans/zh-Hant instead zh-CN?

File: docs/.vitepress/config.ts [hunk 1]
```diff
@@ -24,14 +24,40 @@ export default defineConfig({
...
+        nav: nav(),
+        sidebar: {
+          '/guide/': sidebarGuide(),
+          '/reference/': sidebarReference()
+        }
+      }
+    },
+    zh: {
+      label: '简体中文',
+      lang: 'zh-CN',
```
Comment (taozuhong) [inline docs/.vitepress/config.ts:42]: zh-Hans maybe good and will cover more people.

**Comment by vanchKong** [comment]
> I don't think we need two translation PRs because our goal is the same one. @vanchKong Could you please add me and @shellRaining collaborate your repo so that we can opt the translations conveniently?
> 
> I will close my PR at an appropriate time.

Done!
sorry for the late reply

**Comment by awxiaoxian2020** [comment]
@vanchKong Please fix typo in title, it's `translation`.

**Comment by vanchKong** [comment]
What's the plan about this pr? @brc-dd

**Comment by awxiaoxian2020** [comment]
The PR will be considered when the VitePress is at release stage.@vanchKong

**Comment by brc-dd** [comment]
Yeah, like @awxiaoxian2020 said. It'll be merged as we are close to v1.

**Comment by brc-dd** [comment]
Just a tip -- run `npx simple-git-hooks` -- it will add prettier to your git hooks, and CI won't fail because of that then.

**Comment by brc-dd** [comment]
Hi guys, thanks a lot for maintaining this! Is this ready to merge or are there any blockers/TODOs? If you guys are fine, we can merge this by this week. (we are planning to release v1 in few days)

**Comment by awxiaoxian2020** [comment]
@brc-dd I will check these files tomorrow entirely, and then feel free to merge it!

cc: @vanchKong @shellRaining

**Comment by awxiaoxian2020** [comment]
@brc-dd I have checked these files to ensure the translations line by line with the English content, and we have over 10000 rows which is not perfect but can be improved by contributors easily. So on my side, This PR can be merged now.

**Pull request merged**